File: HSG_HAI_DUONG_22_23/contestants/nguyen_the_khoi/bai1.py
from math import *
with open ("bai1.inp","r") as f:
    a=int(f.readline())
    A=float(f.readline())
    h=int(f.readline())
A=radians(A)
s=h+tan(A)*a
s=round(s,4)
with open ("bai1.out","w") as f:
    f.write(f"{s:.3f}")
    f.close()

File: HSG_HAI_DUONG_22_23/contestants/huynh_minh_thuan/BAI1.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ifstream inp("BAI1.inp");
    ofstream out("BAI1.out");

    int a, h;
    double alpha;

    inp >> a >> alpha >> h;

    double ans = h + a * tan(alpha * 3.14159265359 / 180.0);

    out << fixed << setprecision(3) << ans;

    inp.close();
    out.close();

    return 0;
}
